File: teste_escribo.dart
main() {
  const int numeroSelecionado = 10;

  int somatorioDeValoresDivisiveisPor3e5(int numero) {
    int resultado = 0;

    for (int i = 3; i < numero; i++) {
      if (i % 3 == 0 || i % 5 == 0) {
        resultado += i;
      }
    }
    return resultado;
  }

  print(somatorioDeValoresDivisiveisPor3e5(numeroSelecionado));
}

File: teste_escribo.js
const numeroSelecionado = 11;

function somatarioDeValoresDivisiveisPor3e5(numero) {
    let resultado = 0;

    for (let i = 3; i < numero; i++) {
        if (i % 3 == 0 || i % 5 == 0) {
            resultado += i;
        }
    }
    return resultado;
}

console.log(somatarioDeValoresDivisiveisPor3e5(numeroSelecionado));